### ISSUE TITLE
uniform maker (0.10.7)  not installing in Windows 10

### ISSUE THREAD

**Issue by geskobe** [eriqjaffe/OOTP-Uniform-Maker#79]
Eriq,

Is there a trick to installing your uniform maker (0.10.7) and logo maker (0.2.3) utilities in Windows 10? I have downloaded both the installer and portable versions for X64 and the installation will not start for either of these programs. I do have an earlier version of logo maker installed. I did have v0.10.6 of the uniform maker installed and it worked fine, but I uninstalled that version when I was notified of the update before I tried to install the newer version.

Thanks much.

**Comment by eriqjaffe** [comment]
The only thing should be having to click through the Windows SmartScreen warnings, but that's detailed on the project page.  I just installed the latest releases in a fresh Windows 10 machine without issue - same for the portable versions.

If you're not getting the SmartScreen popups, this may help:  https://www.advancedinstaller.com/prevent-smartscreen-from-appearing.html

**Comment by geskobe** [comment]
Eriq,

Thank you for replying so quickly. I don't know why these programs would not install. However, I did find a workaround. I was able to install them using Windows 8 compatibility settings after running "troubleshoot compatibility" from the Windows context menu. After that, they installed just fine and are now running without any problems in Windows 10. Very strange.

In any event, they are working. 

Thank you for these very nice programs and your support.
